gwpy/astro/range.py

```diff
@@ -483,21 +650,24 @@ def range_spectrogram(hoft, stride=None, fftlength=None, overlap=None,
     # set frequency limits
     f = hoft.frequencies.to('Hz')
     fmin = units.Quantity(
-        rangekwargs.pop('fmin', hoft.df),
+        rangekwargs.pop('fmin', hoft.df.value),
```

Comment:
Again here can we keep using `hoft.df` just in case the units don't match?

Reply:
@duncanmmacleod any plan to change this?

Reply:
bump